types/alias.go

```diff
@@ -27,7 +27,6 @@ const (
 	BlocksPerWeek        = util.BlocksPerWeek
 	BlocksPerMonth       = util.BlocksPerMonth
 	BlocksPerYear        = util.BlocksPerYear
-	BlocksPerEpoch       = util.BlocksPerEpoch
```

Comment:
Removing blocksPerEpoch seems to be a big decision, also one that is quite different from the stated purpose of this pull request. Consider moving to another pull request to avoid future confusion.